packages/features/tasker/tasker.ts

```diff
@@ -14,6 +16,7 @@ type TaskPayloads = {
   triggerFormSubmittedNoEventWebhook: z.infer<
     typeof import("./tasks/triggerFormSubmittedNoEvent/triggerFormSubmittedNoEventWebhook").ZTriggerFormSubmittedNoEventWebhookPayloadSchema
   >;
+  translateEventTypeDescription: TranslateEventTypeDescriptionPayload;
```

Comment:
Nit for consistency (not blocking tho)

```suggestion
  translateEventTypeDescription: z.infer<
    typeof import("./tasks/translateEventTypeDescription").ZTranslateEventTypeDescriptionPayloadSchema
  >;
```

Reply:
addressed